net-mgmt/lldpd/Makefile

```diff
@@ -1,6 +1,6 @@
 PLUGIN_NAME=		lldpd
-PLUGIN_VERSION=		1.1
-PLUGIN_REVISION=	2
+PLUGIN_VERSION=		1.2
+PLUGIN_REVISION=	3
```

Comment:
```suggestion
```

revision is increased on changes without a version change, revision  is cleared on version change